### DISCUSSION TITLE
Registering for Enterprise Edition requiers External user?

### DISCUSSION THREAD

**Discussion by GD-Dal** [openobserve/openobserve#4748] [Q&A] [answered]
Hi.
I have installed OO in a docker container, but I understand there is an enterprise edition of the self hosted docker container as well?
But when I try to sign in at https://auth1.openobserve.ai/ui/login/loginname, there is only possible to use external users to log in?

Is that mandatory to register for the enterprise edition, or am I missing something?

Thanks

**Comment by Manazsharma** [comment]
Hi! 
For Running the self-hosted Enterprise version, it is not mandatory to register with us if you are ingesting within the 200GB/day limit, but you need to contact us sometime soon to register if you ingest more data.  Also, you should be able to log in as an internal user.  You can try exploring our High Availablity mode if it suits your use case. Refer to https://openobserve.ai/docs/ha_deployment/ for further help.

**Comment by GD-Dal** [comment]
Thanks. But I still don't understand how I unlock the enterprise edition and its features in my docker installment.

Thanks again

**Comment by hengfeiyang** [comment, marked as answer]
You can directly use the enterprise version tag:
https://gallery.ecr.aws/zinclabs/openobserve-enterprise

Or, you can use our helm to deploy and enable the enterprise in the yaml:
```
enterprise:
    enable: true
```
